shells/chrome/src/backend.js

```diff
@@ -15,15 +15,8 @@ var BananaSlugBackendManager = require('../../../plugins/BananaSlug/BananaSlugBa
 var Bridge = require('../../../agent/Bridge');
 var inject = require('../../../agent/inject');
 var setupHighlighter = require('../../../frontend/Highlighter/setup');
-var setupRNStyle = require('../../../plugins/ReactNativeStyle/setupBackend');
 var setupRelay = require('../../../plugins/Relay/backend');
 
-// TODO: check to see if we're in RN before doing this?
-setInterval(function() {
```

Comment:
This is Chrome shell. We can't be in RN, and Chrome shell hasn't work on for a long time.